requirements/deploy.txt

```diff
@@ -137,7 +137,7 @@ django-cors-headers==4.7.0
     # via -r requirements/pip.txt
 django-crispy-forms==1.14.0
     # via -r requirements/pip.txt
-django-csp==3.8
+django-csp==4.0
```

Comment:
We need to take a look at this changelog just in case.

Reply:
Yeah, it requires extra work. There is a migration guide at https://django-csp.readthedocs.io/en/latest/configuration.html#migrating-from-django-csp-3-8

Reply:
Opened https://github.com/readthedocs/readthedocs.org/issues/12084 for this.